### PR TITLE
Added Homepage with Routing

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
Added Homepage 
1. Routing
2. Header
3. Footer

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
-      <Main/>
+      <Routes>
+        <Route path="/" element={<Main />} />
+        <Route path="/products/:category" element={<ProductListing />} />
+        <Route path="/product" element={<Product />} />
+        <Route path="/wishlist" element={<WishList />} />
+        <Route path="/cart" element={<Cart />} />
+        <Route path="/signup" element={<SignUp />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/mockman" element={<MockAPI/>} />
```
Comment (shubhamkumar648) [inline src/App.js]: suggestion: you can also make one file for routes and then import route file to main for better readability

**Review by shubhamkumar648 — APPROVED** [review]
consider my comment as a suggestion, good breakdown of component!

File: src/App.js (hunk 1 above)
Comment (SimranjitKaurMaan) [reply src/App.js]: Thanks for the suggestion. Resolved it.

File: src/components/customComponents/HeaderNavBar/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+export const HeaderNavBar = () => {
+    return <><header>
+    <div className="logo-container">
+        <a href="./index.html">
+            <h1>Artly<span className="logo-text">Store</span></h1>
```
Comment (raman1999) [inline src/components/customComponents/HeaderNavBar/index.js:8]: change a href to "Link to" in react router dom

File: src/components/customComponents/HeaderNavBar/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    </div>
+    <a className="btn btn-primary btn-header-login" href="./login.html">Login</a>
+    <div className="header-nav">
+        <nav>
+            <ul>
+                <li>
+                    <div className="badge-container">
+                        <span className="badge badge-md badge-text">1</span>
+                        <a href="./wishlist.html"> 
+                            <img className="img-xs" src="https://res.cloudinary.com/duddwta8d/image/upload/v1648081716/heart_vhmlcf.svg" alt="wishlist"/>
```
Comment (raman1999) [inline src/components/customComponents/HeaderNavBar/index.js:25]: same here

**Review by raman1999 — APPROVED** [review]
code looks good clean to me , good folder structure made

File: src/components/customComponents/HeaderNavBar/index.js (hunk 2 above)
Comment (SimranjitKaurMaan) [reply src/components/customComponents/HeaderNavBar/index.js:8]: Yes done.

File: src/components/customComponents/HeaderNavBar/index.js (hunk 3 above)
Comment (SimranjitKaurMaan) [reply src/components/customComponents/HeaderNavBar/index.js:25]: Done.

**Pull request merged**